services/thermal_mgr/thermal_mgr.h

```diff
@@ -5,7 +5,7 @@
 
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
-  
+  OS_HANDLER_EVENT,
```

Comment:
nit: prefix with `THERMAL_MGR_EVENT_`. When things are defined in header files, we try to be careful with the naming so it doesn't conflict with definitions in other files.